examples/service-clients/azure-client/external-controller/src/controller.ts

```diff
@@ -87,9 +58,9 @@ export class DiceRollerController
 
 	public readonly roll = (): void => {
 		const rollValue = (Math.floor(Math.random() * 6) + 1) as DieValue;
-		this.props.set(diceValueKey, rollValue);
+		this.dice.value = rollValue;
 
 		// Also notify the caller of the local roll (local value setting).
-		this.onSet(rollValue);
+		this.onChanged(rollValue);
```

Comment:
I know this wasn't added in this PR, but at some point the presence example should probably be split out to its own thing (could be interesting to have a presence-only diceroller).  Otherwise the double-tracking gets a little confusing.